src/components/Books.js

```diff
@@ -0,0 +1,110 @@
+import BookState from './BookState';
+
+export default function Books() {
+  return (
+    <ul className="book">
+      <li className="book">
+        <div className="left-content">
+          <div className="book-info">
+            <h4 className="book-category">Action</h4>
+            <h2 className="book-title">The Hunger Games</h2>
+            <h6 className="book-author">Suzanne Collins</h6>
+            <BookState />
+          </div>
+        </div>
+
+        <div className="right-content">
+          <div className="circular-progress-container">
+            <div className="circular-progress" />
+
+            <div className="progress-stat">
+              <p className="percent-complete">64%</p>
+              <p className="completed">Completed</p>
+            </div>
+          </div>
+
+          <div className="current-chapter-container">
+            <div>
+              <p className="current-chapter-label">CURRENT CHAPTER</p>
+              <p className="current-chapter">Chapter 17</p>
+            </div>
+            <div>
+              <button className="btn-progress" type="button">
+                UPDATE PROGRESS
+              </button>
+            </div>
+          </div>
+        </div>
+      </li>
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md) we should create a component that contains individual book state, as well as a button to delete the book 

Kindly consider creating a component for this block of codes inside of the `/components` directory.

Then we can reuse it for the individual book states. 👍🛠